pages.es/common/dnsx.md

```diff
@@ -0,0 +1,38 @@
+# dnsx
+
+> Un equipo de herramientas DNS rápido y multipropósito para ejecutar múltiples consultas DNS.
+> Nota: la entrada a `dnsx` necesita ser pasada a través de `stdin` (pipe `|`) en algunos casos.
+> Ver también: `dig`, `dog`, `dnstracer`.
+> Más información: <https://github.com/projectdiscovery/dnsx>.
+
+- Consulta el registro A de un (sub)dominio y muestra la [re]spuesta recibida:
+
+`echo {{ejemplo.com}} | dnsx -a -re`
+
+- Consulta todos los registros DNS (A,AAAA,CNAME,NS,TXT,SRV,PTR,MX,SOA,AXFR,CAA):
+
+`dnsx -recon -re <<< {{ejemplo.com}}`
```

Comment:
```suggestion
`dnsx -recon -re <<< {{ejemplo.com}}`
```

Not all shells support [_here-strings_ (i.e. `<<<`)](https://www.gnu.org/software/bash/manual/bash.html#Here-Strings). Maybe `echo {{ejemplo.com}} | dnsx -recon -re` would replace the current example.

Would it be worth documenting shell syntax for command examples in `common` to aim for portability @kbdharun @kant?

Reply:
> Would it be worth documenting shell syntax for command examples in `common` to aim for portability @kbdharun @kant?

Yeah

Reply:
Definitively yes!